corehq/apps/user_importer/validation.py

```diff
@@ -171,7 +181,10 @@ class NewUserPasswordValidator(ImportValidator):
     def validate_spec(self, spec):
         user_id = spec.get('user_id')
         password = spec.get('password')
-        if not user_id and not is_password(password):
+        is_account_confirmed = spec_value_to_boolean_or_none(spec, 'is_account_confirmed')
+
+        # explicitly check is_account_confirmed against False because None is the default
+        if not user_id and not is_password(password) and is_account_confirmed is not False:
             return self.error_message
```

Comment:
should this error message change?

Reply:
Do you mean like, change to ""New users must have a password set *unless is_account_confirmed is False*."?

I think that's probably an unnecessary detail to include - particularly at this stage and given that this workflow will be behind a feature flag for a while.

(In case it's not clear, all this does is *allow* new users to be created without a password if `is_account_confirmed == False`. Previously it was never allowed.)

Reply:
https://github.com/dimagi/commcare-hq/pull/26864/commits/64e1d9de6f31bef106e4122d2fe10c47b8de5f46